browser_history/generic.py

```diff
@@ -87,9 +87,13 @@ def __init__(self, plat: utils.Platform = None):
             self.profile_dir_prefixes.append("*")
 
     def profiles(self) -> typing.List[str]:
-        """Returns a list of profile directories
+        """Returns a list of profile directories. If the browser is supported on the current
+        platform but is not installed an empty list will be returned
         :rtype: list(str)
         """
+        if not os.path.exists(self.history_dir):
+            utils.logger.info('{} browser is not installed'.format(self.name))
```

Comment:
You should probably use `logging`'s own style of formatting ([reference](https://docs.python.org/3/howto/logging.html#logging-variable-data)).
(`pylint` also complains of this)

Reply:
I have fixed that and updated the docstring in `fetch` as well.